internal/host/static/repository_host_test.go

```diff
@@ -604,6 +606,10 @@ func TestRepository_UpdateHost(t *testing.T) {
 			assert.NoError(err)
 			require.NotNil(orig)
 
+			set := TestSets(t, conn, catalog.GetPublicId(), 1)[0]
+			TestSetMembers(t, conn, set.PublicId, []*Host{orig})
+			want_SetIds := []string{set.PublicId}
```

Comment:
Variables should be camel case.

```suggestion
			wantSetIds := []string{set.PublicId}
```